game-server/data/static_data/global_drops/rules/open_worlds/rules_map_heiron.xml

```diff
@@ -1594,6 +1594,18 @@
 			<gd_item id="152012033"/><!-- Akairun's Horn -->
 		</gd_items>
 	</gd_rule>
+	<gd_rule rule_name="Named Design" chance="100">
```

Comment:
These designs are not supposed to drop. They can be bought from npcs such as https://aioncodex.com/5x/npc/802413

Reply:
This was just an example. You don't have to be level 65 to buy the designs, which would be pointless indeed.
There are more npcs selling it, even one in Sanctum, according to the "Sold by NPC" tab here: https://aioncodex.com/5x/item/152200508/?sl=1
I also found one in Divine Fortress that is not listed on Aioncodex (don't remember the ID right now).

Reply:
Again, this was just exemplary. I originally searched the 4.8 client data for the merchants and found multiple per design. What I did not fully validate which ones should be spawned, as it was just a very quick initial search. So I just mentioned the first one I could find an existing spawn position in the emulator for.
In my opinion, if there are merchants that can be accessed at level <=50, custom drops should not be added.
I can do a complete search in the client files and PTS files later.